3-networks/envs/shared/hierarchical_firewall.tf

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module "hierarchical_firewall_policy" {
+  source = "../../modules/hierarchical_firewall_policy/"
```

Comment:
It may be worthwhile considering adding this as a submodule of https://github.com/terraform-google-modules/terraform-google-network

@bharathkkb what do you think?

Reply:
yes I think thats a good idea, this will be a general ask outside of foundations too.

Reply:
I can take care of that.